content/doc/administration/requirements/java.adoc

```diff
@@ -30,7 +30,7 @@ See link:/redirect/issue-tracker[our Issue tracker] for known Java compatibility
 
 ## Executing jobs on Jenkins
 
-Jenkins jobs may be executed on Java versions different from the controller/agent runtime.
+You may execute Jenkins jobs on Java versions different from the controller/agent runtime.
```

Comment:
I would revert this block other changes look like improvements, the tense is changed here and I think it was correct before

Reply:
Done